UI / Design Landing Page

bumsyalao opened the pull request:
#### What does this PR do?
- Design of UI components

#### Description of Task to be completed?
- Create components.
- Style components.

#### How should this be manually tested?
n/a

#### Any background context you want to provide?
- App was without void and style.

#### What are the relevant pivotal tracker stories?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
n/a

bumsyalao committed:
feature: Create UI component

- create style.css for styling component
- created header component
- create input component
- create button component

[Delivers #3]

bumsyalao committed:
feature: create-UI-components

- created todoItem and todoList component

bumsyalao committed:
Feature: Design-UI-Components

- Redesign UI components
- Added Todo-Item and Todo-List

BolajiOlajide commented on client/assets/style.css:
```diff
@@ -0,0 +1,119 @@
+.item-color {
+  margin-top: 5%;
+  color: #D5DCDA;
+}
```
Add new line here.

BolajiOlajide commented on client/component/Button.jsx:
```diff
@@ -0,0 +1,11 @@
+  }
+}
+
+export default Button;
```
new line needed here.

BolajiOlajide commented on client/index.js:
```diff
@@ -1,8 +1,12 @@
 
 ReactDOM.render(
-  <h1> Hello I'm working </h1>,
+  <div>
```
No need for this.

bumsyalao committed:
chore: requested-changes

- change Header component to functiona component
- delete npm-debug.log, add to gitignore
- add new line to Button.jsx
- remove comments from style.css
